work_items/kyc-vcs/requirements-to-accept-a-kyc-vc.md

```diff
@@ -0,0 +1,576 @@
+<p>
+Nationality of the Person, Country using the <a href="https://www.iso.org/iso-3166-country-codes.html">ISO</a> 3 Alpha Character country abbreviation
+   </td>
+  </tr>
+  <tr>
+   <td>Gender
+   </td>
+   <td>This is not typically required in certain countries due to gender descrimination (UDAAP for example), but is required in Asia and other parts of the world. 
+<p>
+<strong>Note:</strong> While https://schema.org/Male and https://schema.org/Female may be used, text strings are also acceptable for people who do not identify as a binary gender.
+   </td>
+  </tr>
+  <tr>
+   <td>Account Type
+   </td>
+   <td>We may need this field to determine the type of account being opened or n/a if no account is being opened and this is a one-off transaction.
+<p>
+We can use Schema.org references for <a href="https://schema.org/docs/financial.html">financial extension hierarchy</a> under FinancialProduct Service, but this does not cover all different account types possible.
+   </td>
+  </tr>
+  <tr>
+   <td>Purpose of Account
+   </td>
+   <td>Standardize possible responses to include:
+<ul>
+
+<li>deposits,
+
+<li>checking,
+
+<li>savings,
+
+<li>investing,
+
+<li>sending/receiving money,
+
+<li>currency conversion,
+
+<li>cryptocurrency trading
+</li>
+</ul>
+   </td>
+  </tr>
+  <tr>
+   <td>Occupation - occupationalCategory
+   </td>
+   <td>Suggest we use the <a href="https://isco-ilo.netlify.app/en/isco-08/">ISCO-08</a> International Standard Classification of Occupations either at the sun-major group or minor group level to reduce the possible values
+<p>
+<span style="text-decoration:underline;">Example of ISCO-08:</span>
+<p>
+
+
+<img src="assets/isco-08-example.png" alt="ISCO 08 Example" title="ISCO 08 Example">
+
+   </td>
+  </tr>
+  <tr>
+   <td>Employer Name (“worskfor” in Schema.org)
+   </td>
+   <td>Free text field. Note: this is only required by certain financial institutions and can be hard to validate.
+   </td>
+  </tr>
+  <tr>
+   <td>Source of Funds/Wealth
+   </td>
+   <td>Standardize possible responses include:
+<ul>
+
+<li>employment,
+
+<li>self-employment, 
+
+<li>investment, 
+
+<li>savings, 
+
+<li>retirement, 
+
+<li>gift/inheritance, or 
+
+<li>other (which must come with an explanation in free text)
+</li>
+</ul>
+   </td>
+  </tr>
+  <tr>
+   <td>Account Activity
+   </td>
+   <td>Free text field. Expected transaction amount or volume information is typically asked to assess risk and comply with regulatory obligations.
+   </td>
+  </tr>
+</table>
+
+
+Note, that the contents of this type of credential relies on existing and accepted global data schema’s to enhance consistency and interoperability.
+
+To obtain the information contained within the KYC Verifiable Credential, the Financial Institution (or Verifier) will specify the credential schemas and types they accept via a [Presentation Request](https://identity.foundation/presentation-exchange/#presentation-request).
```

Comment:
Should we define Presentation Request above?

Reply:
will add